packages/excalidraw/animated-trail.ts

```diff
@@ -0,0 +1,195 @@
+import { LaserPointer } from "@excalidraw/laser-pointer";
+import { AnimationFrameHandler } from "./animation-frame-handler";
+import { AppState } from "./types";
+import { getSvgPathFromStroke, sceneCoordsToViewportCoords } from "./utils";
+import type App from "./components/App";
+
+const DECAY_TIME = 200;
+const DECAY_LENGTH = 10;
+
+function easeOutCubic(t: number) {
+  return 1 - Math.pow(1 - t, 3);
+}
```

Comment:
if quadratic works, then let's use `easeOut()` from `packages/excalidraw/utils.ts`. If it doesn't work, let's use it anyway and parametrize the exponent?